docs/api.md

```diff
@@ -120,6 +123,14 @@ object.some_event = t.unreached_func("some_event should not fire");
 Keep in mind that other tests could start executing before an Asynchronous
 Test is finished.
 
+Finally, for cases where you want to simply pass the test when a callback is
+called, there is `step_func_done`, which will generate a step function that
+calls `done()`. For example:
+
+```js
+object.some_event = t.step_func_done();
```

Comment:
Note that `step_func_done` can also take a callback argument; in that case
```js
t.step_func_done(f)
```
is roughly equivalent to
```js
t.step_func(() => {
  f();
  t.done();
})
```